internal/fs/inode/file.go

```diff
@@ -279,11 +315,12 @@ func (f *FileInode) DecrementLookupCount(n uint64) (destroy bool) {
 // LOCKS_REQUIRED(f.mu)
 func (f *FileInode) Destroy() (err error) {
 	f.destroyed = true
-
-	if f.content != nil {
+	if f.localFileCache {
+		cacheObjectKey := &contentcache.CacheObjectKey{BucketName: f.bucket.Name(), ObjectName: f.name.objectName}
+		f.contentCache.Remove(cacheObjectKey)
```

Comment:
Why not calling this?
```
f.content.Destroy()
```

Reply:
f.contentCache.Remove() calls Destroy on the tempfile retrieved from the map.